docs/source/daml/intro/2_Scenario.rst

```diff
@@ -0,0 +1,146 @@
+
+- Getting parties
+- Submitting transactions
+- Creating contracts
+- Testing for failure
+- Archiving contracts
+- Viewing ledger and final ledger state
+
+For this section, start a new module called ``Intro_2_Scenario`` and copy across the ``Token`` contract from :doc:`1_Token`. Alternatively, use the :download:`supplied source file <daml/Intro_2_Scenario.daml>`.
+
+.. scenario_basics:
+
+Scenario Basics
+---------------
+
+A ``Scenario`` is like a recipe for a test, in which the submission of transactions by parties, as well as some external information like party identities and ledger time can be scripted.
+
+Below is a basic scenario that creates a ``Token`` for a party called "Alice".
+
+.. literalinclude:: daml/Intro_2_Scenario.daml
+  :language: daml
+  :start-after: -- TOKEN_TEST_1_BEGIN
+  :end-before: -- TOKEN_TEST_1_END
+
+A ``Scenario`` is declared as a top-level variable and introduced using ``scenario do``. ``do`` always starts a block, so the rest of the scenario is indented.
+
+Before you can create any ``Token`` contracts, you need some parties on the test ledger. In the above scenario, the function ``getParty`` is used to put a party called "Alice" in a valiable ``alice``. There are two things of note there:
+
+1. Note of the use of ``<-`` instead of ``=``. The reason for that is that depending on the ledger the scenario is running on, ``getParty`` may have to look up a party identity or create a new party. ``getParty`` is an ``Action`` that can only be performed once the ``Scenario`` is run in the context of a ledger and ``<-`` means "run the action and bind the result". More on ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
+
+   If that doesn't quite make sense yet, for the time being you can think of this arrow as extracting the right-hand-side value from the ledger and storing it into the variable on the left.
+2. The argument ``"Alice"`` to ``getParty`` does not have to be enclosed in brackets. Functions in DAML are called using the syntax ``fn arg1 arg2 arg3``.
+
+With a variable ``alice`` of type ``Party`` in hand, you can submit your first transaction. Unsurprisingly, this is done using the ``submit`` function. ``submit`` takes two arguments: A Party and an ``Update``. Just like ``Scenario`` is a recipe for a test, ``Update`` is a recipe for a transaction. ``create Token with owner = alice`` is an ``Update``, which translates to the transaction creating a ``Token`` with owner Alice. You'll learn all about the syntax ``Token with owner = alice`` in :doc:`3_Data`.
+
+It's possible to write ``submit alice (create Token with owner = alice)``, but just like scenarios, updates can be assembled using ``do`` blocks. A ``do`` block always takes the value of the last statement within it so the syntax shown in the scenario above gives the same result, whilst being easier to read.
```

Comment:
Passive to active, and some emphasis - you might disagree, I just feel this is a crucial point. 

```suggestion
You could write this as ``submit alice (create Token with owner = alice)``, but just like scenarios, you can assemble updates using ``do`` blocks. A ``do`` block always **takes the value of the last statement within it** so the syntax shown in the scenario above gives the same result, whilst being easier to read.
```